src/state/transactions/updater.tsx

```diff
@@ -118,13 +136,21 @@ export default function Updater(): null {
                 dispatch(updateBlockNumber({ chainId, blockNumber: receipt.blockNumber }))
               }
             } else {
+              checkPrivateTxStatusFailed(hash).then((isFailed) => {
+                // Set transaction status as failed or dropped
```

Comment:
hm maybe we can add a field to `TransactionDetails` to track this status?

https://github.com/Uniswap/interface/blob/main/src/state/transactions/reducer.ts#L15

interface TransactionDetails {
  receipt?: Serializable..
  privateTransactionStatus: 'idle' | 'failed'
}

or something like that, then we can add a new action in the slice to set private tx status?

cc @moodysalem